expression/builtin_time_vec_generated.go

```diff
@@ -1634,44 +1634,8 @@ func (b *builtinTimeTimeTimeDiffSig) vectorized() bool {
 }
 
 func (b *builtinAddDateStringStringSig) vecEvalTime(input *chunk.Chunk, result *chunk.Column) error {
-	n := input.NumRows()
-	unit, isNull, err := b.args[2].EvalString(b.ctx, chunk.Row{})
-	if err != nil {
-		return err
-	}
-	if isNull {
-		result.ResizeTime(n, true)
-		return nil
-	}
-
-	intervalBuf, err := b.bufAllocator.get(types.ETString, n)
-	if err != nil {
-		return err
-	}
-	defer b.bufAllocator.put(intervalBuf)
-	if err := b.vecGetIntervalFromString(&b.baseBuiltinFunc, input, unit, intervalBuf); err != nil {
-		return err
-	}
-
-	if err := b.vecGetDateFromString(&b.baseBuiltinFunc, input, unit, result); err != nil {
-		return err
-	}
-
-	result.MergeNulls(intervalBuf)
-	resDates := result.Times()
-	for i := 0; i < n; i++ {
-		if result.IsNull(i) {
-			continue
-		}
-		resDate, isNull, err := b.add(b.ctx, resDates[i], intervalBuf.GetString(i), unit)
-		if err != nil {
-			return err
-		}
-		if isNull {
-			result.SetNull(i, true)
-		} else {
-			resDates[i] = resDate
-		}
+	for i := 0; i < input.NumRows(); i++ {
+		result.AppendNull()
 	}
```

Comment:
Why return Nulls here?
This is not compatible with its row-based implementation.